### DISCUSSION TITLE
Internal Server error when trying to link to an external Data source

### DISCUSSION THREAD

**Discussion by ALaa13** [Budibase/budibase#5307] [Help] [answered]
Hi, I'm trying Budibase for the first time.
We saw a good usage for Budibase where we can link our database from Airtable into Budibase and use it to get external input from clients. 
However when trying to link Airtable and view the data on a screen in the design tab. Internal Server error keeps showing up. 
I tried to use REST instead of Airtbale to test it. Same error message, However, if I choose Budibase as the Data source (AKA the User table) no error is shown. Could someone explain to me if it's on my end and what I'm doing wrong. or it's on Budibase's end.
Error message:
<img width="824" alt="2" src="https://user-images.githubusercontent.com/28536553/162338808-6d14a506-024b-47a0-b6d0-49aca014c4e3.png">

Airtable setup:
<img width="488" alt="0" src="https://user-images.githubusercontent.com/28536553/162338872-83b4a9f0-5710-42d7-82e4-7b360d977c97.png">

Note: when I query using Airtbale. in the JSON body, it only shows the first record! but when I navigate to the preview tab, It shows all  data
 
<img width="494" alt="1" src="https://user-images.githubusercontent.com/28536553/162338958-41edac52-a0e6-4e96-b1f0-736116e5da17.png">

**Comment by melohagan** [comment, marked as answer]
Hi @ALaa13 

Thank you for your interest in budibase!

I have been able to recreate your issue. It appears to be a UI glitch in that the error is incorrectly shown, but from what I can see there is no issue with the data being returned.

I have a raised a bug here: https://github.com/Budibase/budibase/issues/5318

**Reply by ALaa13** [reply]
Thank you @melohagan  for replying. I'm glad you were able to figure out the issue.
